### ISSUE TITLE
"Light IMDb Ratings Update" Add-on

### ISSUE THREAD

**Issue by viking2** [bkury/skin.night#16]
In the default skin (Estuary), I can run the "Light IMDb Ratings Update" Add-on by going to Add-ons and clicking on the icon. However, nothing happens when I use the Night skin and click on the icon.

EDIT: I just realized that there is no mouse support. I works when I select the icon and hit Enter. 
A minor difference though;
 In Estuary it shows Media scanned with scan progress in %. With Night, it only shows Media scanned and no % Progress.

**Comment by bkury** [comment]
Yes, there is no progress bar yet. It's already on my todo list and will be implemented with the next update.

Also, proper mouse support is already in the works. It's just a very tedious job to do with KODI.

**Comment by viking2** [comment]
OK. Thanks!
